lib/applicants/field-resolvers.ts

```diff
@@ -18,7 +19,11 @@ export const applicantApplicationsResolver: Resolver<Applicant> = async (
  * @returns Array of permit objects
  */
 export const applicantPermitsResolver: Resolver<Applicant> = async (parent, _args, { prisma }) => {
-  return await prisma.permit.findMany({ where: { applicantId: parent?.id } });
+  return await prisma.applicant
+    .findUnique({
+      where: { id: parent.id },
+    })
+    .permits();
```

Comment:
Does this work better in general? Just wondering if there are any tradeoffs between the two approaches.

Reply:
yeah it's a prisma workaround to avoid n+1 queries. For some reason prisma doesn't batch findMany queries but it batches findUnique so this is the easiest way to not make a separate db query for every permit when we can just make 1 

https://www.prisma.io/docs/guides/performance-and-optimization/query-optimization-performance#solving-the-n1-problem

Reply:
Oh that's a great catch! I'll note this down for our backlog when we do optimizations